Geometry/HcalTowerAlgo/src/CaloTowerHardcodeGeometryLoader.cc

```diff
@@ -16,6 +21,15 @@ std::auto_ptr<CaloSubdetectorGeometry> CaloTowerHardcodeGeometryLoader::load(con
   theHFEtaBounds   = m_hcons->getEtaTableHF();
   theHBHEEtaBounds = m_hcons->getEtaTable();
 
+#ifdef DebugLog
```

Comment:
@emiglior - it would be better to use MessageLogger for debug messages, or comment this out.